### ISSUE TITLE
TreeVisibility.Recurse not working correctly in test tree view

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#114251]
Returned here: https://github.com/microsoft/vscode/blob/fd0a3a12e7bff10378f74d3ea2cf16c9ddd354f5/src/vs/workbench/contrib/testing/browser/testingExplorerView.ts#L493

Recurse is acting just like "Visible". I see it getting called here, which doesn't look like it does anything with the "recurse" result:

https://github.com/microsoft/vscode/blob/fd0a3a12e7bff10378f74d3ea2cf16c9ddd354f5/src/vs/base/browser/ui/tree/abstractTree.ts#L544-L564

**Comment by joaomoreno** [comment]
@connor4312 Does it repro if you set the `simpleKeyboardNavigation: true` option?

**Comment by connor4312** [comment]
It does. Though I need to make changes to pass that down, since the WorkbenchObjectTree doesn't pass it down to the parent tree view implementation, and it doesn't look like any other tree view specifies this option...

If I understand correctly, I think setting `simpleKeyboardNavigation` is actually appropriate for _this_ use case, since we use the input box instead of tree view filtering. Though it is still a bug that Recurse can't be used with non-simply-navigated tree views.

**Comment by joaomoreno** [comment]
> If I understand correctly, I think setting simpleKeyboardNavigation is actually appropriate for this use case, since we use the input box instead of tree view filtering. Though it is still a bug that Recurse can't be used with non-simply-navigated tree views.

Yeah. Let's use `simpleKeyboardNavigation` here. IMO let's wait for an actual non-simply-navigated recurse use case. Unless you're feeling adventurous!

**Comment by connor4312** [comment]
Maybe, I will backlog this for now :)
